testapp/react-native.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  dependencies: {
+    'react-native-leveldown': {
+      platforms: {
+        android: null, // disable Android platform, other platforms will still autolink if provided
```

Comment:
Hm, could you help me follow why clients must disable auto linking for this library? Is the idea that this module must be manually linked, rather than auto-linked, because of the leveldb-android dependency?

Reply:
Correct, I had some problems with auto-linking, because of the nested structure of local dependencies.

I managed to publish the fork of `leveldb-android`. Therefore we don't have to include it as a local dependency anymore, it is fetched automatically by Gradle. 
After the last commit a79a6cc, auto-linking works and none of the instructions [above](https://github.com/andymatuschak/react-native-leveldown/pull/4#issuecomment-674019209) are needed!

Reply:
Fantastic!!